src/Synolia/Bundle/FavoriteBundle/Entity/Favorite.php

```diff
@@ -6,87 +6,58 @@
 
 use Doctrine\ORM\Mapping as ORM;
 use Oro\Bundle\CustomerBundle\Entity\Ownership\FrontendCustomerUserAwareTrait;
-use Oro\Bundle\EntityBundle\EntityProperty\CreatedAtAwareTrait;
 use Oro\Bundle\EntityBundle\EntityProperty\DatesAwareInterface;
 use Oro\Bundle\EntityBundle\EntityProperty\DatesAwareTrait;
-use Oro\Bundle\EntityConfigBundle\Metadata\Annotation\Config;
+use Oro\Bundle\EntityConfigBundle\Metadata\Attribute\Config;
 use Oro\Bundle\EntityExtendBundle\Entity\ExtendEntityInterface;
 use Oro\Bundle\EntityExtendBundle\Entity\ExtendEntityTrait;
 use Oro\Bundle\OrganizationBundle\Entity\OrganizationAwareInterface;
 use Oro\Bundle\OrganizationBundle\Entity\Ownership\OrganizationAwareTrait;
 use Oro\Bundle\ProductBundle\Entity\Product;
-use Synolia\Bundle\FavoriteBundle\Model\ExtendFavorite;
 
-/**
- * @ORM\Entity(repositoryClass="Synolia\Bundle\FavoriteBundle\Entity\Repository\FavoriteRepository")
- * @ORM\Table(name="sy_favorite")
-
- * @Config(
- *      defaultValues={
- *          "ownership"={
- *              "organization_field_name"="organization",
- *              "organization_column_name"="organization_id",
- *              "frontend_owner_type"="FRONTEND_CUSTOMER",
- *              "frontend_owner_field_name"="customerUser",
- *              "frontend_owner_column_name"="customer_user_id",
- *          },
- *          "security"={
- *              "type"="ACL",
- *              "group_name"="commerce"
- *          },
- *      }
- * )
- * @ORM\HasLifecycleCallbacks
- */
+#[ORM\Entity(repositoryClass: 'Synolia\Bundle\FavoriteBundle\Entity\Repository\FavoriteRepository')]
+#[ORM\Table(name: 'synolia_favorite')]
```

Comment:
you need a v1_1 that rename the existing table and test the upgrade between 5.1 and 6.0

Reply:
Updated